# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/mips64/pkg_gcc.diff
# Copyright (C) 2020 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-13.2.0/gcc/config/mips/t-linux64.vanilla	2023-10-12 11:53:51.254325005 +0200
+++ gcc-13.2.0/gcc/config/mips/t-linux64	2023-10-12 11:55:59.449328367 +0200
@@ -16,8 +16,8 @@
 # along with GCC; see the file COPYING3.  If not see
 # <http://www.gnu.org/licenses/>.
 
-MULTILIB_OPTIONS = mabi=64/mabi=32
-MULTILIB_DIRNAMES = 64 32
+MULTILIB_OPTIONS = mabi=n32/mabi=32
+MULTILIB_DIRNAMES = n32 32
 MULTILIB_OSDIRNAMES = \
-	../lib64$(call if_multiarch,:mips64$(MIPS_EL)-linux-gnu$(MIPS_SOFT)) \
+	../lib32$(call if_multiarch,:mips64$(MIPS_EL)-linux-gnuabi64$(MIPS_SOFT)) \
 	../lib$(call if_multiarch,:mips$(MIPS_EL)-linux-gnu$(MIPS_SOFT))
